crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -1576,6 +1576,14 @@ impl From<&TransactionKind> for TransactionBlockKind {
                 };
                 TransactionBlockKind::ConsensusCommitPrologueTransaction(consensus)
             }
+            TransactionKind::ConsensusCommitPrologueV2(x) => {
+                let consensus = ConsensusCommitPrologueTransaction {
+                    epoch_id: x.epoch,
+                    round: Some(x.round),
+                    timestamp: DateTime::from_ms(x.commit_timestamp_ms as i64),
+                };
+                TransactionBlockKind::ConsensusCommitPrologueTransaction(consensus)
```

Comment:
I currently didn't create a ConsensusCommitPrologueV2Transaction in graphql rpc. Not sure if we need to (since the digest is mainly for internal purpose? But I do see a need to show it in query for debugging purpose). Even if we do, I think I need to do it in a separate PR given that the amount of work to make graph ql work with digest is non trivial (at least based on my current attempt).

Reply:
It's fine to leave it as-is, thanks @halfprice, but could you add a TODO comment? I'll pick it up and add the digest after you've landed this change.